ibm_catalog.json

```diff
@@ -217,23 +217,30 @@
               "role_crns": [
                 "crn:v1:bluemix:public:iam::::role:Administrator"
               ],
-              "service_name": "iam-identity",
-              "notes": "[Optional] Required if Cloud automation for account configuration is enabled."
+              "service_name": "All Account Management services",
+              "notes": "[Optional] Administrator access is required to create and delete resource groups."
```

Comment:
Does [this](https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vpc/blob/main/ibm_catalog.json#L115) fit well here ?